### ISSUE TITLE
Problem runnign with Python3.8

### ISSUE THREAD

**Issue by adityaghantasala** [parikls/mem_usage_ui#7]
hi 
This tool has been very helpful. Thank you for developing it. 
I am having trouble running your tools with Python3.8

I am running linux Mint 20
I installed the tool using pip3 
It installed the following 
mem_usage_ui in /home/aditya/.local/lib/python3.8/site-packages (1.0.1)
psutil==5.4.8 in /home/aditya/.local/lib/python3.8/site-packages (from mem_usage_ui) (5.4.8)
uvloop==0.12.0rc1 in /home/aditya/.local/lib/python3.8/site-packages (from mem_usage_ui) (0.12.0rc1)
aiohttp==3.5.1 in /home/aditya/.local/lib/python3.8/site-packages (from mem_usage_ui) (3.5.1)
chardet<4.0,>=2.0 in /usr/lib/python3/dist-packages (from aiohttp==3.5.1->mem_usage_ui) (3.0.4)
multidict<5.0,>=4.0 in /home/aditya/.local/lib/python3.8/site-packages (from aiohttp==3.5.1->mem_usage_ui) (4.7.6)
yarl<2.0,>=1.0 in /home/aditya/.local/lib/python3.8/site-packages (from aiohttp==3.5.1->mem_usage_ui) (1.5.1)
async-timeout<4.0,>=3.0 in /home/aditya/.local/lib/python3.8/site-packages (from aiohttp==3.5.1->mem_usage_ui) (3.0.1)
attrs>=17.3.0 in /usr/lib/python3/dist-packages (from aiohttp==3.5.1->mem_usage_ui) (19.3.0)
idna>=2.0 in /usr/lib/python3/dist-packages (from yarl<2.0,>=1.0->aiohttp==3.5.1->mem_usage_ui) (2.8)

Now when I run the tool from the terminal I get the following error

Traceback (most recent call last):
  File "/home/aditya/.local/bin/mem_usage_ui", line 5, in <module>
    from mem_usage_ui.__main__ import main
  File "/home/aditya/.local/lib/python3.8/site-packages/mem_usage_ui/__init__.py", line 5, in <module>
    import uvloop
  File "/home/aditya/.local/lib/python3.8/site-packages/uvloop/__init__.py", line 7, in <module>
    from .loop import Loop as __BaseLoop  # NOQA
  File "uvloop/includes/stdlib.pxi", line 114, in init uvloop.loop
AttributeError: module 'sys' has no attribute 'set_coroutine_wrapper'

Any Idea why this is the case. 

Best

**Comment by allyo-dmytro-smyk** [comment]
@adityaghantasala hi, thanks for the feedback! Issue is related to some deprecations happened in python3.8, so i need to update the requirements. Will try to do this shortly.

As a temp solution you can upgrade both `uvloop` and `aiohttp` packages and run the `mem-usage-ui`.
e.g.
```bash
pip3.8 install --upgrade uvloop
pip3.8 install --upgrade aiohttp
```

**Comment by adityaghantasala** [comment]
Hi @allyo-dmytro-smyk  Thank you for your quick response. I tried to upgrade uvloop and aiohttp. It works with that trick. Currently I have 

```
ERROR: mem-usage-ui 1.0.1 has requirement aiohttp==3.5.1, but you'll have aiohttp 3.6.2 which is incompatible.
ERROR: mem-usage-ui 1.0.1 has requirement uvloop==0.12.0rc1, but you'll have uvloop 0.14.0 which is incompatible.
```

Those two errors but still mem_usage_ui works.
Thanks again.

**Comment by parikls** [comment]
@adityaghantasala fixed in `1.0.3`
